freeipa.spec.in

```diff
@@ -142,7 +148,8 @@
 %if 0%{?fedora} < 37
 %global bind_version 9.11.24-1
 %else
-%global bind_version 32:9.18.7-1
+# last BIND version without backport of DNSSEC support over OpenSSL provider API
+%global bind_version 32:9.18.35-2
```

Comment:
Fedora 41 is still supported but provides only [bind-9.18.33-1.fc41](https://koji.fedoraproject.org/koji/buildinfo?buildID=2653315) 
This line would break the build on f41.

Reply:
I'll drop down the dependency here and will bump on `-encrypted-dns` subpackage so that we make sure it gets proper one to install. We cannot run full encrypted DNS support in F41 anyway.